src/lib/render.jsx

```diff
@@ -1,5 +1,7 @@
 var redux = require('redux');
 var thunk = require('redux-thunk').default;
+// JSX syntax transforms to React.createElement
+var React = require('react'); // eslint-disable-line
```

Comment:
When the JSX below (`<StoreProvider store={store}>...`) gets transpiled into regular JS, the JSX syntax  `<Component .../>` becomes `React.createElement(Component ...)`, so even though we don't explicitly use React in this file, it's required due to the JSX syntax. There's no way currently to tell eslint that JSX syntax == React requirement (see https://github.com/eslint/eslint/issues/1534), so I needed to disable linting on this line.  Previously since React was an external, we didn't actually need to ever require react to use it, except for the fact that eslint would complain.